deploy/kubernetes/rbac.yaml

```diff
@@ -51,7 +51,7 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 
 ---
-# Attacher must be able to work with config map in current namespace
```

Comment:
Also add a comment that a driver only needs to include one of these rules depending on which leader election method it chose